<ui version="4.0" >
 <class>QChooseGUIStyle</class>
 <widget class="QDialog" name="QChooseGUIStyle" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string comment="Window title" >Choose a GUI style</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Available styles :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QListWidget" name="stylesList" >
       <property name="horizontalScrollBarPolicy" >
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScrollMargin" >
        <number>17</number>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QPushButton" name="tryButton" >
         <property name="text" >
          <string>Try selected style</string>
         </property>
         <property name="shortcut" >
          <string>Alt+T</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="keepButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Keep style</string>
         </property>
         <property name="shortcut" >
          <string>Alt+K</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>125</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="font" >
          <font>
           <italic>false</italic>
           <underline>true</underline>
          </font>
         </property>
         <property name="text" >
          <string>Current Style</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentStyleLabel" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>125</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton" >
         <property name="text" >
          <string>Quit</string>
         </property>
         <property name="shortcut" >
          <string>Alt+Q</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tryButton</tabstop>
  <tabstop>keepButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
